### DISCUSSION TITLE
My github account got flagged

### DISCUSSION THREAD

**Discussion by mirrorleechdo** [community/community#50781] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello github support my github account got flagged I don't why I got flagged plz help me to un flag it

**Comment by radar-blip** [comment, marked as answer]
For account related issues, see here: [Support page](https://support.github.com/contact/account)

**Reply by queenofcorgis** [reply]
Support is the only avenue for rectifying account flags as @linuxlawson mentioned above. The Community is unable to assist or escalate so I'll be locking this post. Best of luck!

**Comment by nao-chan-dev** [comment]
I have the same problem. Have you tried to make a new account to see if it changes anything?
